src/FluentlyHttpClient/GraphQL/GqlResponse.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Net.Http;
+
+namespace FluentlyHttpClient
```

Comment:
only the extensions method class should be **without** the GraphQL, so this one should be `FluentlyHttpClient.GraphQL`